iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -34,60 +35,51 @@ class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableVi
         tableView.delegate = self
         tableView.dataSource = self
         
-        
-        /******** START EXAMPLE ***********/
+        // Set up a bar button item to toggle debug info on background app refresh
+        let item = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(MBArticlesViewController.showTimestamps))
+        self.navigationItem.setRightBarButton(item, animated: false)
         
         // 1. the managed context has to be passed in (UIApplication should only be accessed from main thread)
         guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
             print("Unable to get the app delegate!")
             return
         }
-        let managedContext = appDelegate.persistentContainer.viewContext
-        /*
-        // 2. Sync all the Data
-        MBStore().syncAllData(context: managedContext) { (err: Error?) in
-            if let syncErr = err {
-                print(syncErr)
-                return
-            }
-            
-            print("Sync Success!")
-            
-            print("\nAUTHOR NAMES")
-            print(MBStore().getAuthors(managedContext: managedContext).map { $0.name })
-            print("\nAUTHOR IDS")
-            print(MBStore().getAuthors(managedContext: managedContext).map { $0.authorID })
-            
-            print("\nCATEGORIES")
-            print(MBStore().getCategories(managedContext: managedContext).map { $0.name })
+        
+        guard let managedContext = appDelegate.persistentContainer?.viewContext else {
+            print("Unable to get the managed object context!")
+            return
+        }
+  
+        
+        let oneWeekAgoTimestamp = Date().timeIntervalSinceReferenceDate - MBConstants.SECONDS_IN_A_WEEK
+        let lastUpdateTimestamp = UserDefaults.standard.double(forKey: MBConstants.DEFAULTS_KEY_ARTICLE_UPDATE_TIMESTAMP)
+        
+        if lastUpdateTimestamp > oneWeekAgoTimestamp {
```

Comment:
This is where I feel like we don't necessarily need to be doing this kind of update in the view controller. I don't know exactly where to put this, maybe in the App Coordinator instead in its start method. I think it's ok for now because we are still figuring out this syncing stuff but down the road might want to change it

Reply:
agreed. I will clean this up once I solve the random crashing issue